scripts/find-broken-links

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# example usage: $ find-broken-links
+#  it's setup to use 1.11 docs Urls
+#  it will print the links that won't return 200 to stdout: <link> <http-status-code>
+
+files=$(git ls-files)
+for file in $files
+do
+  # skip files with links we don't care about
+  matchesNonCodeFile=$(echo $file | grep -Eo "(\.json|\.gif|\.png|\.svg|-test|-cy|\.jpg|\.icns|\.ico|\.iconsproj|\.ttf|\.woff)")
```

Comment:
@ahoskins would make more sense to be only if `matchesCodeFile` that way only `.js`. Not a big deal though

Reply:
Yeah, I thought about inverting this. The thing is, there are other file formats we care about too. Like .md, .html, .ts (maybe in the future :smile:)?

Reply:
Got it, in that case I think it's fine because I thought was only `.js` so makes sense.